### PR TITLE
Added secondary header

### PR DESCRIPTION
**Pull request by ruthvik947**
Some CSS inconsistencies exist. Modify if/when possible:
@lorsposto @luuj @titusjung

### CODE REVIEW

**Comment by ruthvik947** [comment]
Also I unintentionally amended to previous commit

File: src/main/webapp/banana/css/styles.css [hunk 1]
```diff
@@ -9,32 +9,40 @@ html, body {
 
 .usc-header {
 	padding-top: 7px;
-	padding-bottom: 7px;
+	/*padding-bottom: 7px;*/
```
Comment (chrismattmann) [inline src/main/webapp/banana/css/styles.css]: remove if commented out.

**Comment by chrismattmann** [comment]
please remove commented code and then +1 to merge @ruthvik947

**Comment by lorsposto** [comment]
@ruthvik947 oh did you add in the headers too? thanks! address above and merge!

**Pull request merged**